README.Rmd

```diff
@@ -22,13 +22,156 @@ knitr::opts_chunk$set(
+
+```{r read_protti, message=FALSE, warning=FALSE, eval = FALSE}
+# Load data
+data <- read_protti("filename.csv")
+```
+
+Since we will actually use synthetic data for this example we are going to call the `create_synthetic_data` function from protti. Of course you do not need to do this step in your analysis pipeline. 
+
+The data this function creates is similar to data obtained from a LiP-MS experiment. Please note that any of the steps in this workflow can also be applied to protein abundance data that contains protein IDs and protein intensities.
+
+```{r synthetic_data, message=FALSE, warning=FALSE}
+set.seed(42) # Makes example reproducible
+
+# Create synthetic data
+data <- create_synthetic_data(n_proteins = 100,
+                              frac_change = 0.05,
+                              n_replicates = 4,
+                              n_conditions = 2,
+                              method = "random_effect",
+                              additional_metadata = FALSE)
+
+# The method "random_effect" as opposed to "dose-response" just randomly samples the change of significantly 
+# changing peptides for each condition. They do not follow any trend and can go in any direction.
+```
+
+#### Clean and Normalise Data
+
+Before you start analysing your data it is recommended that you filter out any observations not necessary for your analysis. These include for example: 
+
+* **Decoys**: Sometimes decoys are included in reports and it is advised to filter them out before data analysis. 
+* **Contaminants**: MaxQuant includes contaminant information in their reports. They are useful for quality control purposes but should be filtered out for analysis.
+* **Non-Proteotypic Peptides**: For the analysis of LiP-MS data you should not include non-proteotypic peptides in your analysis since it is not possible to make sense of them in case they change significantly. 
+
+On your own data you can easily achieve this with dplyrs `filter` function. Our synthetic data does not require any filtering at this step. 
+
+Due to the fact that with increasing raw intensities also variances increase, statistical tests would have a bias towards lower intense peptides or proteins. Therefore you should log2 transform your data to correct for this mean-variance relationship. We do not need to do this for the synthetic data as it is already log2 transformed. For your own data just use dplyrs `mutate` together with `log2`. 
+
+In addition to filtering and log2 transformation it is also advised to median normalise your data to equal out small differences in overall sample intensities that result from unequal sample concentrations. protti provides the `median_normalisation` function for this purpose. This function generates an additional column called `normalised_intensity_log2` that contains the normalised intensities.
```

Comment:
You could explain here that this method is based on Spectronaut's median normalisation and that you only have to use it if your search engine has not already median normalised the data.